images/win/scripts/Installers/Windows2016/Install-SSDT.ps1

```diff
@@ -5,8 +5,9 @@
 
 Import-Module -Name ImageHelpers -Force
 
-#SSDT for Visual Studio 2017 (15.9.3)
-$InstallerURI = 'https://download.microsoft.com/download/5/2/D/52DEF429-5B17-470C-82D9-654116080B15/SSDT-Setup-ENU.exe'
+#SSDT for Visual Studio 2017
+#The link down below points to the latest version of SSDT for Visual Studio 2017
+$InstallerURI = 'https://go.microsoft.com/fwlink/?linkid=2124518'
```

Comment:
What if this link is changed?

Reply:
@miketimofeev , https://go.microsoft.com/fwlink is symlink that always points to the latest version and will be updated if new version is released